test/e2e/composables.test.ts

```diff
@@ -0,0 +1,45 @@
+import { fileURLToPath } from 'node:url'
+import { isWindows } from 'std-env'
+import { join } from 'pathe'
+import { expect, test } from './test-utils'
```

Comment:
is there any chance we can use a nuxt runtime test rather than an e2e test?

Reply:
@huang-julien that would be nice

Reply:
great :fire: !